src/components/UserInfo/UserInfo.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { Button, Container, Box } from '@material-ui/core';
+import UserStatistics from './UserStatistics/UserStatistics';
+import UserSubmissions from './UserSubmissions/UserSubmissions';
+import UserTeams from './UserTeams/UserTeams';
+import UserContests from './UserContests/UserContests';
+import UserEditProfile from './UserEditProfile/UserEditProfile';
+import UserInventory from './UserInventory/UserInventory';
+import styles from './UserInfo.css';
+
+class UserInfo extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            display: 'statistics'
+        };
+
+        this.handleClick = this.handleClick.bind(this);
+    }
+
+    handleClick(event) {
+        const id = event.target.getAttribute('id');
+        if(id === 'statistics') this.setState({display: 'statistics'});
+        else if(id === 'submissions') this.setState({display: 'submissions'});
+        else if(id === 'teams') this.setState({display: 'teams'});
+        else if(id === 'contests') this.setState({display: 'contests'});
+        else if(id === 'inventory') this.setState({display: 'inventory'});
+        else if(id === 'editProfile') this.setState({display: 'editProfile'});
+    }
```

Comment:
Você pode refatorar isso aqui pra ficar assim:
`const id = event.target.getAttribute('id');
 this.setState({display: [id]});`

Reply:
this.setState({display: id});